### DISCUSSION TITLE
Support for new Figma Variables

### DISCUSSION THREAD

**Discussion by exhtml** [d01000100/figma-token-engine#54] [Ideas]
Hi folks! 
Are you planning tu support new Figma Variables into FTE?
Thank you very much!

**Comment by d01000100** [comment]
I've been looking into it since they released. You can see my WIPs on #53, but it's still on the exploration phase.

Sadly, the Figma API is just for Enterrpise level accounts and I only have Organization tier, so it's halted for the time being. 

Very probably I'll also add an input for Variables exported and "pasted" from a plugin or something similar.

**Comment by d01000100** [comment]
Check out the latest #62 v0.3.0 release!
